x-pack/metricbeat/module/gcp/metrics/_meta/docs.asciidoc

```diff
@@ -45,6 +45,8 @@ services under "Google Cloud metrics", but does not work for other services
 (`kubernetes` aka GKE for example).
 This option allow to override the default and specify an arbitrary metric prefix.
 
+* *location_label*: Use this option to specify the resource label that identifies the location (such as zone or region) for a Google Cloud service when filtering metrics. For example, labels like `resource.label.location` or `resource.label.zone` are used by Google Cloud to represent the region or zone of a resource.
```

Comment:
Specify if required or not .
Also provide available values that it can get

Reply:
Sorry to re-open that, could you specify if it's required or not? If not required, what is the default value?